### PR TITLE
Use correct path to x86 installers to ANCM

### PR DESCRIPTION
**Pull request by jkotalik**
Right now, the x86 msis for ANCM contain the wrong bitness for dlls. This addresses it.

### CODE REVIEW

File: src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs [hunk 1]
```diff
@@ -162,25 +162,43 @@
...
+                    <File Id="AspNetCoreModuleDll"
+                        Name="aspnetcorev2.dll"
+                        Source="$(var.AspNetCoreV2.TargetPath)"
+                        DiskId="1"
+                        Vital="yes">
+                    </File>
+                <?else ?>
+                    <File Id="AspNetCoreModuleDll"
+                        Name="aspnetcorev2.dll"
+                        Source="$(var.AspNetCoreV2WoW64.TargetPath)"
```
Comment (joeloff) [inline src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs]: Why don't you just define a single variable in the .wixproj or pass it through using -d, then you don't need to duplicate the File elements.

File: src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs [hunk 2]
```diff
@@ -162,25 +162,43 @@
         <Directory Id="INSTALLLOCATION" ShortName="ANCM" Name="$(var.ProductName)">
           <Directory Id="VersionDir" Name="$(var.ProductVersionString)">
             <Component Id="AspNetCoreModule" Guid="84ed6ce6-c8a3-4fa8-a872-c98a1d15dd4f" Win64="$(var.IsWin64)">
```
Comment (joeloff) [inline src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs]: Is there a reason why the component GUIDs aren't generated and why you have the same GUIDs for both 64-bit and 32-bit components?

Comment (jkotalik) [reply src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs]: You can't install the 64 and 32 bit msis on the same machine, so we should be okay in that scenario.

Why GUIDs aren't generated is a different issue that should probably be resolved.

File: src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs (hunk 1 above)
Comment (jkotalik) [reply src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs]: Right now these are defined via project references here: https://github.com/aspnet/AspNetCore/blob/master/src/Installers/Windows/AspNetCoreModule-Setup/Directory.Build.props#L45. In the wixproj file, can I just use $(AspNetCoreV2.TargetPath)?

Comment (jkotalik) [reply src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs]: I tried a few things and nothing worked, ideas?

Comment (joeloff) [reply src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs]: if you have a .wixproj, you can do something like <DefineConstants>$(DefineConstants);Foo=$(SomePath)</DefineConstants>

Then in your .wixproj, you'll be able to access $(var.Foo). If you conditionally set $(SomePath) on the architecture you'll have one constant defined for both bitness

Comment (jkotalik) [reply src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs]: Went with an approach where I define the constants in the wxs file instead (as the variables I wanted were defined in DefineConstants).

**Review by joeloff — APPROVED** [review]

**Comment by jkotalik** [comment]
Verified this too.

**Pull request merged**